en/micro-integrator/docs/use-cases/tutorials/file-processing.md

```diff
@@ -249,7 +250,9 @@ Package the artifacts in your composite application module to be able to deploy
 
 ### Step 4: Configure the Micro Integrator server
 
-1.  Click the <b>Embedded Micro Integrator Configuration</b> icon on the upper menu to open the dialog box.
+1.  Click the <b>Embedded Micro Integrator Configuration</b> (<img src="../../../assets/img/tutorials/common/server-config-64x64.png" width="20">) icon on the upper menu to open the dialog box.
+
+    <img src="../../../assets/img/tutorials/file-processing/embedded-server-configs.png" width="500">
 
 2.  Add the following server configurations (to the `deployment.toml` file) using the upper section in the dialog box.
```

Comment:
Please add the folder path to the deployment.toml file. [MI-HOME]/conf/deployment.toml

Reply:
We are adding the configurations from the tool itself. There a new panel in Integration Studio that can be used to add the configurations and libs.